plugin.json

```diff
@@ -19,6 +19,13 @@
     "header": "",
     "footer": "",
     "settings": [
+      {
+        "key": "enableSyncJob",
+        "display_name": "Enable sync job",
+        "type": "bool",
+        "help_text": "When false, disables the sync job that creates and maintains chat and channel subscriptions",
+        "default": true
```

Comment:
Could we make this match the `disableCheckCredentials` pattern, and default to `false` instead?

Reply:
The current behaviour is that by default the sync job is enabled.  I don't want to change the default least it mess up something's existing instance by turning it off after an upgrade.

Reply:
Right, that's why I'm proposing `disableSyncJob` with a default of `false`. (We had to do this for `disableCheckCredentials` since I couldn't rely on the `plugin.json`'s defaults /actually/ taking effect.)

Reply:
The setting is flipped.